utils/bmc_stats.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/python
+
+"""
+Sample script to get stats from BMC rest interface.
+
+Assumes that the usb0 interface on BMC is 192.168.0.1 and usb0 interface
+on microserver is 192.168.0.2.
+"""
+
+import pprint
+import urllib2
+import json
+from optparse import OptionParser
+
+apis = [
+  '/api/sys',
+  '/api/sys/mb',
+  '/api/sys/mb/fruid',
+  '/api/sys/bmc',
+  '/api/sys/server',
+  '/api/sys/sensors',
+  '/api/sys/gpios',
+  '/api/sys/modbus_registers',
+  '/api/sys/psu_update',
+  '/api/sys/slotid',
+]
+
+pp = pprint.PrettyPrinter()
+
+def get_api(host, api):
+  print("=== %s ===" % api)
+  url = "http://%s:8080%s" % (host, api)
+  d = urllib2.urlopen(url).read()
+  j = json.loads(d)
+  pp.pprint(j)
+
+def get_all_apis(host):
+  for api in apis:
+    get_api(host, api)
+
+def list_apis():
+  for api in apis:
+    print api
+  print 'all'
+
+
+parser = OptionParser()
```

Comment:
Perhaps drop this into a `if __name__ == '__main__'` incase anyone wanted to build on / import from these.